packages/create-block-interactive-template/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+const { join } = require( 'path' );
+
+module.exports = {
+	defaultValues: {
+		slug: 'example-interactive',
+		title: 'Example Interactive',
+		description: 'An interactive block with the Interactivity API',
+		dashicon: 'interactivity',
```

Comment:
'interactivity' doesn't exist in the dashicon library. Did you mean [media-interactive](https://developer.wordpress.org/resource/dashicons/#media-interactive)?

Reply:
> Did you mean [media-interactive](https://developer.wordpress.org/resource/dashicons/#media-interactive)?
Yes, that's what I meant. I've fixed it 👍